layouts/shortcodes/fields/latinx.html

```diff
@@ -0,0 +1,14 @@
+<div class="field">
+  <label class="label">Do you identify as a Latinx / a person of Latin American descent?*</label>
+  {{ if .Get "help" }}
+  <p class="help">{{ .Get "help" }}</p>{{ end }}
+  <div class="control">
+    <label class="radio">
+      <input type="radio" aria-label="Do you identify as a Latinx?" name="identify_as_latinx_in_tech" value="Yes" required> Yes
```

Comment:
name needs to be changed because that's what netlify uses for form integration with slack

Reply:
so "identify_as_latinx" would suffice

Reply:
updated